packages/mobile/src/tokens/saga.ts

```diff
@@ -18,6 +18,10 @@ import * as utf8 from 'utf8'
 
 const TAG = 'tokens/saga'
 
+// We can safely assume that any balance query returning a number
+// higher than this is incorrect (currently set to 10M)
+export const VALORA_BALANCE_UPPER_BOUND = new BigNumber('1e10')
```

Comment:
nit: we shouldn't use the Valora naming  in the open source code.

```suggestion
export const BALANCE_UPPER_BOUND = new BigNumber('1e10')
```